modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesKernelBoot.java

```diff
@@ -516,7 +510,7 @@ public static void registerMBeans() throws AndesException {
      *
      * @throws AndesException
      */
-    private static void initClusterManager() throws AndesException {
+    private static void initClusterManager() throws AndesException,ClassNotFoundException, IllegalAccessException, InstantiationException {
```

Comment:
Is it really necessary throw these additional exceptions? I think it's problematic if these are thrown from somewhere